src/Uno.Analyzers/Uno.Analyzers.csproj

```diff
@@ -1,76 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
```

Comment:
You'll also need to add the property that disables the use of the targetframework in the output path. The original project  was legacy and did not have it in the path, this will cause the nuget to fail packing.